ic-payments/tests/state_machine.rs

```diff
@@ -109,6 +136,8 @@ mod tests {
         Decode!(&response, Option<Nat>).unwrap()
     }
 
+    // It fails
```

Comment:
Should we fix it?

Reply:
maybe or maybe not. This test was not executed before this PR so I don't know if it ever worked or when it was broken. In addition, we are not using ic-payments anymore so I would not spend time with it